scss/foundation-styles/type.scss

```diff
@@ -42,6 +43,10 @@ h4,
 .h4-heading {
   @include heading("Zilla Slab", 400, 0, $black, $white);
   @include scaleText(22px, 32px, 24px, 36px);
+
+  @media (min-width: $bp-md) {
```

Comment:
Same. Please separate overrides from `/foundation-styles`.
Also to set line-height for type bases, let's utilize the mixin `scaleText`.

@natalieworth - 2 questions

1. From sketch it seems like line height is `30/24=1.25`. Is `1.25` the value we want or `1.2`?
<img width="230" alt="image" src="https://user-images.githubusercontent.com/2896608/57548134-0d877780-7315-11e9-8e0b-c8bbae8f73b4.png">

2. Whatever the line height override we decide to have for `h4`, does it apply to mobile AND desktop? Or just desktop?

Reply:
1. @mmmavis 1.25 works! Then we won't have to update anything in Sketch and the .5 will not really make a difference.

2. I think we can keep if for desktop only right now. If mobile looks weird I can update another brand review/ update ticket to change that.

Reply:
@mmmavis my override files are in the footer pr #1054. Since have to separate them, I'll add this edit there to avoid merge conflicts.

Reply:
If that's the case, let's get the footer PR merged first, pull in changes in `master` into this branch, and update this PR. If PRs have inte-dependancies, we should land them sequentially.